content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -91,73 +102,89 @@ For example:
      "type": "MODIFIED",
      "object": {"kind": "Pod", "apiVersion": "v1", "metadata": {"resourceVersion": "11020", ...}, ...}
    }
-   ...
+   …
    ```
 
-A given Kubernetes server will only preserve a historical list of changes for a limited time. Clusters using etcd3 preserve changes in the last 5 minutes by default.  When the requested watch operations fail because the historical version of that resource is not available, clients must handle the case by recognizing the status code `410 Gone`, clearing their local cache, performing a list operation, and starting the watch from the `resourceVersion` returned by that new list operation. Most client libraries offer some form of standard tool for this logic. (In Go this is called a `Reflector` and is located in the `k8s.io/client-go/cache` package.)
+   {{< note >}}
+   Although the API server responded with a `Content-Type` header indicating a JSON response,
+   the response to a _watch_ request is in fact a stream of multiple documents. For JSON format, the client can
```

Comment:
We may want to decide how to write API request verbs, i.e. _watch_ (`_watch_`), or *watch* (`*watch*`), or **watch** (`**watch**`), or `watch` (\`watch\`).
 
Is it okay to say that the response body is a [JSON lines](https://jsonlines.org/) format?

Reply:
I don't think it is definitely JSONlines, but it might be.

Reply:
I've gone for `**watch**`.